tests/unit/govNotify.unitTest.ts

```diff
@@ -167,6 +163,16 @@ describe('gov-notify', () => {
     });
 
     context('when the request is not valid', () => {
+      it('response should resolve with one batch failure', () => {
+        CertificateDownloadService.prototype.getCertificate = jest.fn().mockResolvedValue('');
+        NotificationService.prototype.sendNotification = jest.fn().mockRejectedValueOnce('error sending with email client');
+        return lambdaTester(handler)
+          .event(event)
+          .expectResolve((result: any) => {
```

Comment:
when the request is not valid, shouldn't this be reject and not resolve?

Reply:
No,I would expect the lambda to succesfully return an array with batch item failures. Errors are no longer thrown (therefore creating rejection) but simply logged, then retried etc